input_example.txt

```diff
@@ -1,4 +1,7 @@
 [example]
-disk = '/home'
-wsize = '1m'
-period = '5'
+/disk=/home
+/wsize=1m
+/period=5
```

Comment:
For maintaining backward compatibility, can we handle the `/` prefix in the code ?